### DISCUSSION TITLE
Record wav or mp3

### DISCUSSION THREAD

**Discussion by pierrehub2b** [airsdk/Adobe-Runtime-Support#3071] [General]
Hello,

Is it possible to record a wav or a mp3 file on the disk from a bytearray captured with the microphone in an Air native desktop application ?

Thank you for any help

Pierre

**Comment by pol2095** [comment]
see https://github.com/MrMitch/Simple-AS3-Recorder
and
https://github.com/cyrildiagne/Shine-MP3-Encoder-on-AS3-Alchemy

**Reply by pierrehub2b** [reply]
Thank you so much for the links ... it works for me now !
